service-python/featuretoggle/src/lib/utils.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+
+
+def create_features_list():
+    """
+    Parse a yaml file into a usable feature flags with boolean values
```

Comment:
Have you given thought to how the real `features.yml` will be deployed in higher environments? Do engineers just edit this file (`service-python/featuretoggle/src/lib/features.yml`) directly, and have those values baked into the Docker image?

And how might this microservice's design allow for turning features on/off while in flight?

Reply:
For now, yes, engineers would edit the features.yml file directly. In the future, a UI connection would toggle and update the values (using another to-be-defined utility function perhaps). I am not quite sure how that works with python versus like, javascript... but that was my thought.